recipes/shadow3/conda_build_config.yaml

```
@@ -0,0 +1,10 @@
+c_compiler_version:            # [unix]
+  - 7                          # [osx]
+  - 7                          # [linux]
+
+cxx_compiler_version:          # [unix]
+  - 7                          # [osx]
+  - 7                          # [linux]
+
+fortran_compiler_version:  # [unix]
+  - 7                      # [unix]
```

I suggest we leave out the OSX configuration and just keep the Linux one with v7 compilers here:
```suggestion
c_compiler_version:            # [linux]
  - 7                          # [linux]

cxx_compiler_version:          # [linux]
  - 7                          # [linux]

fortran_compiler_version:  # [linux]
  - 7                      # [linux]
```